Makefile

```diff
@@ -71,6 +71,20 @@ test-e2e:
 	GINKGO_FOCUS="\[efs-csi\]" \
 	./hack/e2e/run.sh
 
+.PHONY: test-e2e-external-eks
+test-e2e-external-eks:
```

Comment:
this works for me. however if you want to trick the CI into executing it, remove the `test-e2e` rule above and rename this to `test-e2e` because basically what the CI does is run `make test-e2e`. Then in the logs we should be able to validate that an EKS cluster with IMDS disabled passed the test

Reply:
reference: https://github.com/kubernetes/test-infra/blob/5f5d5a6d1130e44c28f4481d47ac00c55ca50f55/config/jobs/kubernetes-sigs/aws-efs-csi-driver/aws-efs-csi-driver-presubmits.yaml#L57-L59

Reply:
never mind it didn't work either, it only worked up until the test itself then I got 

```

  creating file system
  Unexpected error:
      <*errors.errorString | 0xc000557450>: {
          s: "no subnets found for cluster test-cluster-23625-k8s-local",
      }
      no subnets found for cluster test-cluster-23625-k8s-local
  occurred
```

will debug, I guess there could be a bug here https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/5e1fcd3e915d62d3b091c6de780ff9e6816f3a7b/test/e2e/cloud.go#L215-L231

Reply:
I opened https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/707 to try to fix this

Reply:
https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/707 has merged, running the tests via the hack scripts should be fixed now although i see below manual tests also succeeded